utils/docker/Dockerfile.jammy

```diff
@@ -40,6 +40,12 @@ RUN mkdir /ms-playwright && \
     npm i /tmp/playwright-core.tar.gz && \
     npm exec --no -- playwright-core mark-docker-image "${DOCKER_IMAGE_NAME_TEMPLATE}" && \
     npm exec --no -- playwright-core install --with-deps && rm -rf /var/lib/apt/lists/* && \
+    # Workaround for https://github.com/microsoft/playwright/issues/27313
```

Comment:
I'd mention that the plugin was loading soup2, so that we immediately know later.